app/controllers/redirections_controller.rb

```diff
@@ -0,0 +1,13 @@
+class RedirectionsController < ApplicationController
+  def next
+    redirection = Redirection.find_by(slug: params[:slug])
```

Comment:
If a slug doesn't exist, this won't create it -- is that intentional? I was thinking something like this would be cool:

* Anyone adds a `/next` link
* When someone clicks that link we create a record
* No registration required